stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -1014,6 +1014,17 @@ function complete_identifiers!(suggestions::Vector{Completion}, @nospecialize(ff
                 ex = Meta.parse(lookup_name, raise=false, depwarn=false)
             end
             isexpr(ex, :incomplete) && (ex = nothing)
+        elseif isexpr(ex, :call) && length(ex.args) > 1
+            isinfix = s[end] != ')'
+            # A complete call expression that does not finish with ')' is an infix call.
+            if !isinfix
+                # Handle infix call argument completion of the form bar + foo(qux).
+                frange, end_of_identifier = find_start_brace(@view s[1:end-1])
```

Comment:
should this use `prevind` rather than `end-1`?

Reply:
If so, please add a test where the last character is more than 1 byte.

Reply:
Absolutely, thanks for the review!
Done in https://github.com/JuliaLang/julia/pull/51366/commits/5c5b61d8401a22160b81b2ff3b8e40f995d531b9

Reply:
I only caught it because we just fixed the same problem literally 2 days ago https://github.com/JuliaLang/julia/pull/51541